collection-scripts/gather_namespaced_resources

```diff
@@ -100,13 +101,12 @@ for INSTALL_NAMESPACE in $PRODUCT_NAMESPACE $INSTALL_NAMESPACES $MANAGED_FUSION_
           # shellcheck disable=SC2086
           { oc describe ${command_desc} -n ${INSTALL_NAMESPACE}; } >>"${COMMAND_OUTPUT_FILE}"
      done
+     # NOTE: This is a temporary fix for collecting the storagecluster as we are not able to collect the storagecluster using the inspect command
+     { oc get storageclusters -n "${INSTALL_NAMESPACE}" -o yaml; } >"${BASE_COLLECTION_PATH}/namespaces/${INSTALL_NAMESPACE}/oc_output/storagecluster.yaml" 2>&1
+     { oc get storagesystem -n "${INSTALL_NAMESPACE}" -o yaml; } >"${BASE_COLLECTION_PATH}/namespaces/${INSTALL_NAMESPACE}/oc_output/storagesystem.yaml" 2>&1
+     { oc get storageconsumer -n "${INSTALL_NAMESPACE}" -o yaml; } >"${BASE_COLLECTION_PATH}/namespaces/${INSTALL_NAMESPACE}/oc_output/storageconsumer.yaml" 2>&1
 done
 
-# NOTE: This is a temporary fix for collecting the storagecluster as we are not able to collect the storagecluster using the inspect command
```

Comment:
why we have this change from 105-109, why is this shifted inside the loop?

Reply:
+1, did we collect values from "only one" namespace before?

The new code makes much sense but I am asking as to why nobody brought it up earlier?

Regards

Reply:
Seems we didn't notice earlier, but the INSTALL_NAMESPACE has more than one namespace and it should be inside the loop, orelse it wont collect the info. Just a part of cleanup.

Reply:
Just noticed this bug in code while testing for extended namespace.

Reply:
Alright :)

Reply:
> Just noticed this bug in code while testing for extended namespace. 

While you're at it can you see if the `inspect` method works now? If so we can use that instead of the current workaround. 

Regards